students/JoeNunnelley/session04/trigrams.py

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env python3
+"""
+Trigrams Progject
+Author : Joe Nunnelley
+"""
+
+import random
+
+
+def read_file(filename):
+    """Read a file and get its words into a list"""
+    words_in = []
+    with open(filename, 'r') as corpus:
+        words_in = corpus.read().replace('\n', ' ') \
```

Comment:
string.punctuation returns a string of all punctuation characters. You can check to see if the last character (usually where the punctuation is) to replace it with the empty string.